src/controllers/shared.controllers/director.occupation.controller.ts

```diff
@@ -22,7 +22,7 @@ import { occupationErrorMessageKey } from "../../utils/api.enumerations.keys";
 import { CompanyAppointment } from "private-api-sdk-node/dist/services/company-appointments/types";
 import { getCompanyAppointmentFullRecord } from "../../services/company.appointments.service";
 import { STOP_TYPE } from "../../utils/constants";
-import { selectLang, getLocalesService, getLocaleInfo } from "../../utils/localise";
+import { selectLang, getLocalesService, getLocaleInfo, addLangToUrl } from '../../utils/localise';
```

Comment:
any specific reason this import is changed from double quotes to single quotes, looks like IDE auto arrange import ?